source/Client/BugTrap.h

```diff
@@ -160,7 +154,13 @@ typedef enum BUGTRAP_FLAGS_tag
 	  * @brief Automatically restart the application after the crash has been
 	  * handled.
 	  */
-	 BTF_RESTARTAPP    = 0x200
+	 BTF_RESTARTAPP    = 0x200,
+     /**
+      * @brief By default BugTrap show a "More..." button to open
+      * the advanced ui in the simple dialog. Enable this flag
+      * hide the button.
+      */
+     BTF_HIDEMOREBUTTON = 0x400
```

Comment:
We can leave the comma for future possible flags:```suggestion
     BTF_HIDEMOREBUTTON = 0x400,
```

Reply:
ok, I change that.